packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -193,6 +195,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
           isInnerZoneKeystroke={ this._isFocusZoneInnerKeystroke }
         >
           <div className={ styles.screenReaderOnly } role='alert' id='selected-suggestion-alert' aria-live='assertive'>{ selectedSuggestionAlert } </div>
+          <div className={ styles.screenReaderOnly } role='alert' id='suggestion-removed-alert' aria-live='assertive'>{ this.state.removedItemAlertText }</div>
```

Comment:
There definitely shouldn't be two live regions. Selected suggestion should win if it's not null, that means the user has changed something and that should be the only region.